### ISSUE TITLE
Show variable snapshot button is not accessible using the keyboard: A11y_Visual Studio Code Jupyter Extensions_Home_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-jupyter#17074]
### GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #A11ySev1;#DesktopApp; #Visual Studio Code Jupyter Extensions; #GH-VSCodeJupyterExtensions-Win32-Oct2025; #WCAG2.1.1; #FTP; #Keyboard;

### Environment Details:
Application: Visual Studio Code Jupyter Extensions
Version: 1.105.0-insider (user setup)
OS build :26100.4770

### Repro Steps

1. Install "GitHub Copilot app modernization - upgrade for Java" app.
2. Launch the Visual studio code.
3. "Chat" popup will appear. Type "upgrade this project to java 21 using java upgrade tools" in edit field and hit ENTER key.
4. Verify that Show variable snapshot button is not accessible using the keyboard.

### Actual Experience.
Show variable snapshot button is not accessible using the keyboard

### Expected Experience
Show variable snapshot button should be accessible using the keyboard keys.

### User Impact:
Keyboard and screen reader users will not be able to access the controls.

### Attachment

https://github.com/user-attachments/assets/9f8cb190-7072-488c-86f8-dde79322d712

**Comment by meganrogge** [comment]
@amunger is there a command for this?

**Comment by amunger** [comment]
we deprecated our data viewer (and this view should also be eventually replaced with the accessible native variable view), but that icon will delegate to `dataWrangler.openNotebookVariable` when data wrangler is installed.

**Comment by meganrogge** [comment]
So there's not a command?

**Comment by amunger** [comment]
`dataWrangler.openNotebookVariable` can be used to get to this functionality directly.
Closing as we are deprecating this view in favor of the native variable explorer with https://github.com/microsoft/vscode/issues/166296, available with the `notebook.variablesView` setting enabled
